<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diag_pvslot</class>
 <widget class="QDialog" name="diag_pvslot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Slot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>(0) PSP (+ PDA backports)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(1) SEGA crossover</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(2) PDA, Mirai</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(3) Live mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(4) Contests</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(5) Concerts</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(6) f</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(7) F 2nd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(8) X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(9) Dummy PVs</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DoubleDigitQSpinBox" name="spinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>List...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleDigitQSpinBox</class>
   <extends>QSpinBox</extends>
   <header>DoubleDigitQSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>diag_pvslot</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>diag_pvslot</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
